content/docs/command-reference/remote/add.md

```diff
@@ -162,36 +162,27 @@ For more information about the variables DVC supports, please visit
 
 ```dvc
 $ dvc remote add -d myremote azure://mycontainer/path
-$ dvc remote modify --local myremote connection_string 'mystring'
+$ dvc remote modify --local myremote connection_string 'mysecret'
```

Comment:
is it a recommended/default way still? should we skip the credentials setup here?

Reply:
Skip credentials because of [default credential](https://docs.microsoft.com/en-us/python/api/azure-identity/azure.identity.defaultazurecredential) ? Good Q, I have the impression we still want to rec `connection_string` but not sure. WDYT @isidentical?

Reply:
> I have the impression we still want to rec connection_string but not sure. 

Connection string is definitely easier since it usually contains all the relevant info inside (account_name/account_key/sas_token etc)

Reply:
> Connection string is definitely easier since it usually contains all the relevant info inside (account_name/account_key/sas_token etc)

I guess my question here was about the most common situation. E.g. with AWS the most natural way is to expect the credentials file and CLI to be set. And everything else is needed only CI for example (to make secrets self-contained) or other automation scenarios.

Reply:
I assume `connection_string` is the most common method, considering that the dvc 2<= recommended it as the first class method for everyone (since it didn't support things like sas_token as a separate option). Outside of dvc, it seems like account_name/account_key combination is also recommended alongside with connection_string method.

Reply:
I would guess in favor of `connection_string` when considering remote machines (e.g. CI/CD) or other people's envs where not AWS CLI is setup.